packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -296,4 +331,8 @@ class BundleToolArtifact extends CachedArtifact {
   String get storageUrl {
     return 'https://github.com/google/bundletool/releases/download/1.15.6/bundletool-all-1.15.6.jar';
   }
+
+  @override
+  String? get checksum =>
+      '''38ae8a10bcdacef07ecce8211188c5c92b376be96da38ff3ee1f2cf4895b2cb8''';
```

Comment:
I generated this manually, unfortunately Github Releases (where the bundle tool is hosted) does not offer an automatic checksum feature. Since we use hard coded version url, I can't think on any problem in this approach, but let me know if I am missing something.

Reply:
Yeah, that's fine.  This is more useful as a comment in the source than a comment on the PR though. :)

Reply:
For sure, the comment was just to get a discussion going :). Gonna put a comment.